xfce/xfconf/Portfile

```diff
@@ -25,7 +25,7 @@ depends_build   port:xfce4-dev-tools port:intltool port:pkgconfig
 depends_lib     port:libxfce4util \
                 port:dbus port:dbus-glib \
```

Comment:
If you don’t mind, make it more humanly readable (I realize it is pre-existing code, but why not fix it now): mixing vertical and horizontal lists of dependencies is awful.

Reply:
Fixed.  Everything should be good to go now.